modules/webhook-configuring-acs.adoc

```diff
@@ -10,19 +10,19 @@ Create a new integration in {product-title} by using the webhook URL.
 
 .Procedure
 . On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
-. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
-. Click *New Integration* (*`add`* icon).
-. Enter a name for *Integration Name*.
+. Scroll down to the *Notifier Integrations* section and select *Generic Webhook*.
+. Click *New integration*.
+. Enter a name for *Integration name*.
 . Enter the webhook URL in the *Endpoint* field.
-. If your webhook receiver uses an untrusted certificate, enter a CA certificate in the *CA Cert* field; otherwise leave it blank.
+. If your webhook receiver uses an untrusted certificate, enter a CA certificate in the *CA certificate* field; otherwise leave it blank.
```

Comment:
One nit, two suggestions. This is extremely picky and I don't really have a preference whether you take it up or not :D 

"*CA certificate* field; otherwise, leave it blank". 

"*CA certificate* field. Otherwise, leave it blank".